pkg/services/alerting/notifiers/line.go

```diff
@@ -75,6 +75,11 @@ func (this *LineNotifier) createAlert(evalContext *alerting.EvalContext) error {
 	body := fmt.Sprintf("%s - %s\n%s", evalContext.Rule.Name, ruleUrl, evalContext.Rule.Message)
 	form.Add("message", body)
 
+        if evalContext.ImagePublicUrl != "" {
+        form.Add("imageThumbnail", evalContext.ImagePublicUrl)
```

Comment:
IMHO, sending full size image as thumbnail is not a good idea. So how about delete imageThumbnail until grafana support thumbnail image (or you can make a PR to support this feature)